mypy/options.py

```diff
@@ -113,6 +117,9 @@ def __init__(self) -> None:
         self.debug_cache = False
         self.quick_and_dirty = False
 
+        # plugins
+        self.plugins = []  # type: List[PluginRegistry]
```

Comment:
Just for information here is the issue about cycles https://github.com/python/mypy/issues/3054